modules/uhttp/handler.go

```diff
@@ -59,5 +59,7 @@ type handlerWrapper struct {
 // ServeHTTP calls Handler.ServeHTTP(ctx, w, r) and injects a new service context for use.
 func (h *handlerWrapper) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ctx := fxcontext.New(context.Background(), h.host)
+	stopwatch := h.host.Metrics().SubScope("http").SubScope("handle").Timer("sla").Start()
```

Comment:
do not chain subscopes: do `http.handle` in one go so we don't copy stuff around without need